StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/CustomerSessionAdapter/CustomerSessionAdapter.swift

```diff
@@ -107,7 +107,13 @@ extension CustomerSessionAdapter {
         return stripePaymentMethodId
     }
 
-    func fetchSelectedPaymentOption(for customerId: String) -> CustomerPaymentOption? {
+    func fetchSelectedPaymentOption(for customerId: String, elementsSession: STPElementsSession? = nil) -> CustomerPaymentOption? {
+        if configuration.allowsSetAsDefaultPM,
```

Comment:
Nit; I think a guard here is more appropriate as it enforces that you return a value: 
```swift
guard configuration.allowsSetAsDefaultPM,
      let elementsSession = elementsSession,
      let customer = elementsSession.customer,
      let defaultPaymentMethod = customer.defaultPaymentMethod else {
    return CustomerPaymentOption.defaultPaymentMethod(for: customerId)
}

return CustomerPaymentOption.stripeId(defaultPaymentMethod)
```

Reply:
Also, it seems unnecessary to pass in the entire `STPElementsSession` here, we only really need the customer.